blocks/bricks/sequence_generators.py

```diff
@@ -577,14 +735,7 @@ def get_dim(self, name):
 
 
 class FakeAttentionRecurrent(AbstractAttentionRecurrent, Initializable):
-    """Adds fake attention interface to a transition.
-
-    Notes
-    -----
-    Currently works only with lazy initialization (can not be initialized
-    with a single constructor call).
-
-    """
+    """Adds fake attention interface to a transition."""
```

Comment:
Maybe it worth mentioning, that it uses no glimpses and just feeds states and contexts to the transition.